### PR TITLE
test: add ability to test locally

### PR DESCRIPTION
**Pull request by tkanos**
Add the same script than in travis, but in the makefile to be able to test it locally quickly.

### CODE REVIEW

**Comment by tkanos** [comment]
You are 100% right (I miss to work with you), it was changed.

File: Makefile [hunk 1]
```diff
@@ -22,9 +22,5 @@ httpbin-publish: httpbin-container
 	docker push bluehoodie/httpbin
 
 smoke-test: install
```
Comment (colindickson) [inline Makefile]: i would probably call it `test` instead of `smoke-test`.  its cleaner :)

**Comment by colindickson** [comment]
you still planning on coming to the meetup tonight?

File: Makefile (hunk 1 above)
Comment (tkanos) [reply Makefile]: done

**Comment by tkanos** [comment]
Yep, Hope to see you.

**Pull request merged**